vendor/tektelic/t0006377-memo.yaml

```
@@ -0,0 +1,77 @@
+name: TEKTELIC MEMO Multi-Purpose Display Tablet
```

I forgot to mention, please don't put the vendor name in the name of the device.
```suggestion
name: MEMO Multi-Purpose Display Tablet
```